packages/tinytest/tinytest.js

```diff
@@ -203,7 +203,7 @@ export class TestCaseResults {
   // The upshot is, if you want to test whether an error is of a
   // particular class, use a predicate function.
   //
-  throws(f, expected) {
+  throws(f, expected, message) {
```

Comment:
I think we have TS typedef for this, we need to update DeclaredTypes or at least open an issue for this. Also maybe `only` and `onlyAsync` are also missing there.